lib/ansible/plugins/doc_fragments/template_common.py

```diff
@@ -11,8 +11,8 @@ class ModuleDocFragment(object):
 
     # Standard template documentation fragment, use by template and win_template.
     DOCUMENTATION = r'''
-short_description: Template a file out to a remote server
 description:
+- Template a file out to a remote server.
```

Comment:
I don't think this is needed, seems like we just need to remove `short_description` here and just duplicate it across to `template.py` and `win_template.py`.